wolfssl/wolfcrypt/rsa.h

```diff
@@ -215,6 +215,9 @@ struct RsaKey {
 #ifdef WOLF_CRYPTO_CB
     int   devId;
 #endif
+#if defined(WOLF_PRIVATE_KEY_ID) || defined(WOLFSSL_ASYNC_CRYPT)
```

Comment:
Please only wrap this in `HAVE_PKCS11`.

Reply:
All references should now be wrapped with `HAVE_PKCS11`

Reply:
In our configuration (on x86-64), there is a hole at the end of RsaKey, since the last element is a byte (dataIsAlloc). You could put your new flag there so as not to grow the struct size (though I guess the impact might be different on other platforms). Since we're going to be creating a lot of these, if memory usage increase can be minimized, that would surely help.